appcheck/appcheck.go

```diff
@@ -36,6 +36,8 @@ var JWKSUrl = "https://firebaseappcheck.googleapis.com/v1beta/jwks"
 
 const appCheckIssuer = "https://firebaseappcheck.googleapis.com/"
 
+const tokenVerificationUrlFormat = "https://firebaseappcheck.googleapis.com/v1beta/projects/%s:verifyAppCheckToken"
```

Comment:
You are correct: this URL indeed requires a project identifier. What we can do here is separate the domain from the rest of the URL. This way, you can create a constant (maybe `tokenVerifierDomain`?) that would be identical to `appCheckIssuer` (for now), and you can refactor the rest of the path `v1beta/projects/%s:verifyAppCheckToken` inside a new helper function whose only job is to construct the full URL given a project ID.

In my opinion, the reason a helper function is better than a constant for constructing a formatted string is that the format call and the format string are in the same place, which makes the code a lot easier to read and maintain. It also explicitly tells the caller what the format string depends on (as a function parameter).